timbr/DetailsTableViewController.m

```diff
@@ -24,15 +24,59 @@ - (void)viewDidLoad {
     [super viewDidLoad];
     [self.tableView registerNib:[UINib nibWithNibName:@"DetailsTableViewCell" bundle:nil] forCellReuseIdentifier:@"DetailsTableViewCell"];
     [self.tableView registerNib:[UINib nibWithNibName:@"ChartTableViewCell" bundle:nil] forCellReuseIdentifier:@"ChartTableViewCell"];
+    
+    LogCategory *mockLog = [LogCategory getMockLog];
+    self.logCategory =  mockLog;
+    PFQuery *query = [PFQuery queryWithClassName:@"LogCategory"];
+    [query whereKey:@"parent" equalTo:[PFUser currentUser]];
+    [query includeKey:@"entry"];
+    [query findObjectsInBackgroundWithBlock:^(NSArray *categories, NSError *error) {
+        if (categories.count > 0) {
```

Comment:
@ChrisGuzman for some reason above is always empty. If I comment out line 31 - where clause, this gets 4, which makes sense as it fetches all.